src/main/java/uk/gov/companieshouse/model/CreatedBy.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Builder
```

Comment:
Why did't you use builders in other places? I could possibly make your tests more fluent :)

Reply:
i've removed the builder on this class for now to be consistent. I had problems serialising/deserialising the data in my tests tests when using the builder pattern. We can add this later then there's more time